Source/Models/Vehicle.swift

```diff
@@ -4,34 +4,31 @@ public class Vehicle {
     enum State: String {
         case online
     }
-    let color: String
+    let color: String?
     let displayName: String?
     let identifier: Int
-    let optionCodes: [String]
-    let userIdentifier: Int
+    let optionCodes: String
```

Comment:
If `optionCodes` is a comma separated array we should separate the string on all occasions of `,` and set the result to optionCodes.

But this shouldn't be an array, right? It should probably be a set or a dictionary so we can check if a vehicle has an option without having to go iterate through the array.